tests/testthat/test_RT_functions.R

```diff
@@ -18,21 +18,6 @@ test_that('we can create an RT ticket', {
   expect_type(ticket, 'character')
 })
 
-# test_that('create_ticket errors gracefully', {
```

Comment:
Same here - if RT is failing these calls, let's add Github tickets rather than removing the work, and we can discuss what to do.

Reply:
#25